packages/fxa-settings/src/styles/tailwind.css

```diff
@@ -4,4 +4,5 @@
 
 body {
   @apply text-base font-body;
+  @apply bg-grey-50;
```

Comment:
Is this the right grey?

Reply:
It's not, I know you added `-20` in your PR, I'm going to add that here and update it👍